src/auth/init-authentication.ts

```diff
@@ -46,6 +46,10 @@ async function signUpHandler(): Promise<void> {
 
       window.showInformationMessage(authMessage.signedUp)
       ext.outputChannel.appendLine(authMessage.signedUp)
+
+      await iamHelpers.createDefaultProject()
+      ext.explorerTree.refresh()
```

Comment:
Let's move this line to `createDefaultProject()` method after `iamState.clear()`, they're usually called together